tests/RecaptchaTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+class RecaptchaTest extends TestCase
+{
+    public function testRecaptchaFail():void
+    {
+        $response = $this->json('GET', '/contexts/guest/users',
+            [
+                "params" => [
```

Comment:
Could you clarify why we are passing params to the get?

Reply:
To be clear, i see why you are using options, that makes sense, but not sure what the params here are doing